xwiki-platform-core/xwiki-platform-xclass/xwiki-platform-xclass-ui/src/main/resources/XWiki/ClassSheet.xml

```diff
@@ -318,6 +323,46 @@
     [[$services.localization.render('platform.xclass.defaultClassSheet.template.view',
       [$templatePath.trim()]) »&gt;&gt;${classTemplateDoc.fullName}]]
   #end
+  ## Create a template provider only if a template for the current class exists.
+  #if ($classTemplateDoc.getObject(${doc.fullName}))
+    (% id="HClassTemplateProvider" %)
+    = {{translation key="platform.xclass.defaultClassSheet.templateProvider.heading"/}} =
+
+      {{info}}$services.localization.render('platform.xclass.defaultClassSheet.templateProvider.description',
+        ['//']){{/info}}
```

Comment:
why the `//`?

Reply:
If it's for italics, that seems weird.

Reply:
Yes, this is for italics and I used it to be consistent with the existing approach of the sheet and template designed in this page. We can't use style in translation pages since their syntax is `plain/1.0` and I didn't want to duplicate the parameter as I can use the same for wrapping the text (`{0}some italic text{0}`)

Reply:
I don't fully recall our best practice but I remember we had some rules about wiki markup or HTML in translation. Does someone remember? cc @tmortagne @mflorea

Reply:
@vmassol no HTML or wiki syntax in translation messages, that's the best practice I've been using so far.

Reply:
Yes better avoid any kind of syntax as much as possible to avoid trouble. Also in the case of wiki syntax it can create important security holes.

Reply:
> no HTML or wiki syntax in translation messages,

Yes I remember that part. It just seems a bit weird to me that we use parameters to specify wiki syntax. Seems like we're missing something somewhere since that shouldn't be the role of parameters I think.

Reply:
Yes what we are missing is a translation syntax.